.github/workflows/check.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     name: Additional Builds and Concurrency Tests
     env:
       CARGO_TERM_COLOR: always
-      RUSTFLAGS: -D warnings
+      RUSTFLAGS: -D warnings -C target-cpu=native
```

Comment:
so dynamic feature detection does not work? If so, should we enable it in `cargo.toml` as well?

Reply:
On my machine at least, the clmul feature is not available until I tell rust to enable it.

My understanding is that there might be features that are enabled by default, but the default configuration attempts to compile for any CPU with the target architecture, with features disabled if they are not available too all/most/? CPUs with that architecture.

Reply:
Oh yes, a huge difference.  Many more features in the latter than the former.  sse2 is in both, but I need the latter for aes.